src/nifgen/metadata/attributes_addon.py

```diff
@@ -50,6 +50,33 @@
     1050103: { "codegen_method": "no" },  # ERROR_ELABORATION - IVI Attribute - #824
     1050003: { "codegen_method": "no" },  # QUERY_INSTR_STATUS - IVI Attribute - #824
     1050004: { "codegen_method": "no" },  # CACHE - IVI Attribute - #824
+    1150311: { "codegen_method": "no" },  # READY_FOR_START_EVENT_LEVEL_ACTIVE_LEVEL - nifgen review - Remove event properties - #858
```

Comment:
no need to say "nifgen review" as that's captured in commit messages in Git.

Reply:
Removed